src/components/theme/modal/modal.service.tsx

```diff
@@ -87,6 +88,14 @@ class ModalServiceType {
       return this.panel.hide(identifier);
     }
   }
+
+  public setTopInsetsState(state: boolean): void {
```

Comment:
Bad naming. What does it mean to set Top Insets State to true?

Reply:
just applied set/get prefix to `shouldUseTopInsets` variable